app/routes/cdo/edit/activity-confirmation.js

```diff
@@ -14,6 +14,10 @@ const getAlternativeReturnLink = details => {
   return details.source === 'dog' ? `${routes.viewDogDetails.get}/${details.indexNumber}` : 'NOT YET DEFINED'
 }
 
+const getActivityLink = details => {
+  return details.source === 'dog' ? `${routes.viewDogActivities.get}/${details.pk}/dog` : 'NOT YET DEFINED'
```

Comment:
Same comment as above about retaining 'pk' to keep generic for 'owner' too

Reply:
sorted